src/test/java/net/xdn/selenium/pivotaltracker/AccountTest.java

```diff
@@ -0,0 +1,45 @@
+package net.xdn.selenium.pivotaltracker;
+
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Created by Ariel Mattos on 4/12/2017.
+ */
+public class AccountTest {
+
+    public static final String ACCOUNT_NAME = "Test8";
+    public static final String USERNAME = "xaindeneb@gmail.com";
```

Comment:
Don't share accounts in the Pull request
Please use the Environment class for that